src/main/java/mate/academy/spring/service/UserServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.service;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.model.User;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+    private UserDao userDao;
```

Comment:
```suggestion
    private final UserDao userDao;
```
make your code consitent, if you use final for injected dependencies, use it everywhere